specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/subscriptions.json

```diff
@@ -868,6 +1088,179 @@
           "readOnly": true
         }
       }
+    },
+    "PutTenantPolicyRequestProperties": {
+      "description": "Put tenant policy request properties.",
+      "properties": {
+        "blockSubscriptionsLeavingTenant": {
+          "description": "Blocks the leaving of subscriptions from user's tenant.",
+          "type": "boolean"
+        },
+        "blockSubscriptionsIntoTenant": {
+          "description": "Blocks the entering of subscriptions into user's tenant.",
+          "type": "boolean"
+        },
+        "exemptedPrincipals": {
+          "description": "List of user objectIds that are exempted from the set subscription tenant policies for the user's tenant.",
+          "type": "array",
+          "items": {
+            "format": "uuid",
+            "type": "string",
+            "example": "00000000-0000-0000-0000-000000000000"
+          }
+        }
+      }
+    },
+    "TenantPolicy": {
+      "description": "Tenant policy.",
+      "properties": {
+        "policyId": {
```

Comment:
Why is this policyId property required? As this property is not exposed in PUT call, it should be marked as readOnly.

Reply:
Yes Updated

Reply:
The previous feedback was that since we donot have an Id parameter in the URI, we cannot have it in the Id field. The policyId is same as tenantId.